data_parsing/parse.py

```diff
@@ -0,0 +1,63 @@
+import pandas as pd
+
+# Define the column names
+columns = [
```

Comment:
let's convert the column name into the following format- all lower case, connected with `-`, e.g., `entry_name`.